devolo_home_control_api/properties/multi_level_sensor_property.py

```diff
@@ -48,10 +41,9 @@ def unit(self, unit: int):
                  "voltage": {0: "V", 1: "mV"}
                  }
         try:
-            self._unit = units[self.sensor_type].get(unit, str(unit))
+            return units[self.sensor_type].get(self._unit, str(self._unit))
```

Comment:
Can be simplified like this:

```suggestion
            return units.get(self.sensor_type).get(self._unit, str(self._unit)) or str(self.unit)
```

Reply:
No, that will not work as, we try to access get on a NoneType.